BraveRewardsExample/ViewController.swift

```diff
@@ -147,6 +147,16 @@ class ViewController: UIViewController {
   }
 }
 
+extension ViewController: BraveRewardsDelegate {
+  func faviconURL(fromPageURL pageURL: URL, completion: @escaping (URL?) -> Void) {
```

Comment:
This is never called

Reply:
I think this is because we never use `GetActivityInfoFromUrl` or whatever that method is called. Therefore no saveVisitInternal happens with a faviconUrl passed in